### ISSUE TITLE
High training accuracy, results shifted and noisy

### ISSUE THREAD

**Issue by MustafaArikan** [alexgkendall/SegNet-Tutorial#73]
Dear Alex, dear all,

I am training segnet with multi-channel images using lmdb and the data layer.
After a while the training accuracy is quite high (>0.9) and the loss gets low but I am having these strange artifacts in the results.

Top image, GT, Bottom image is one of the results
![grafik](https://cloud.githubusercontent.com/assets/9170096/22232081/831c6922-e1e7-11e6-9432-c97d25f6954e.png)

At first I thought this is a export, shape or data type problem. 
Then I tried to experiment with the learning rate and let segnet train for more epochs, the results seem to shift and there is less noisy structures but I still don't know what the problem is. Do you have any ideas ?

![grafik](https://cloud.githubusercontent.com/assets/9170096/22232157/0ce6ba86-e1e8-11e6-9509-6886b841eb57.png)

**Comment by junaidmalik09** [comment]
did you manage to solve this?

**Comment by MustafaArikan** [comment]
Yes, i had mixed up the x, y dimensions for the input.

**Comment by junaidmalik09** [comment]
Replacing ConvTranspose2D with Upsampling+Conv2D combo solved it for me
